src/components/Projects/WBS/WBSDetail/EditTask/EditTaskModal.jsx

```diff
@@ -654,6 +356,552 @@ const EditTaskModal = props => {
+            aria-label="Search user"
+            placeholder="Link"
+            className="task-resouces-input"
+            data-tip="Add a link"
+            onChange={e => props.setLink(e.target.value)}
+          />
+          <button
+            className="task-resouces-btn"
+            type="button"
+            data-tip="Add Link"
+            onClick={props.addLink}
+          >
+            <i className="fa fa-plus" aria-hidden="true" />
+          </button>
+        </div>
+        <div>
+          {props.links?.map((link, i) =>
+            link.length > 1 ? (
+              <div key={i} className="task-link">
+                <a href={link} target="_blank">
+                  {link.slice(-10)}
+                </a>
+                <span className="remove-link" onClick={() => props.removeLink(i)}>
+                  x
+                </span>
+              </div>
+            ) : null,
+          )}
+        </div>
+      </td>
+    </>
+  );
+};
+
+const TaskCategory = props => {
+  if (!props.canEdit) {
+    return (
+      <>
+        <td scope="col">Category</td>
+        <p>{props.category}</p>
+      </>
+    );
+  }
+  return (
+    <>
+      <td scope="col">Category</td>
+      <td scope="col">
+        <select
+          value={props.category}
+          onChange={e => {
+            props.setCategory(e.target.value);
+          }}
+        >
+          <option value="Housing">Housing</option>
+          <option value="Food">Food</option>
+          <option value="Energy">Energy</option>
+          <option value="Education">Education</option>
+          <option value="Soceity">Society</option>
+          <option value="Economics">Economics</option>
+          <option value="Stewardship">Stewardship</option>
+          <option value="Not Assigned">Not Assigned</option>
+        </select>
+      </td>
+    </>
+  );
+};
+
+const TaskInfo = props => {
+  const editorViewInit = {
+    readonly: true,
+    menubar: false,
+    plugins: 'advlist autolink autoresize lists link charmap table paste help',
+    toolbar: false,
+    branding: false,
+    min_height: 180,
+    max_height: 300,
+    autoresize_bottom_margin: 1,
+  };
+
+  const editorInit = {
+    ...editorViewInit,
+    readonly: false,
+    plugins: 'advlist autolink autoresize lists link charmap table paste help',
+    toolbar:
+      'bold italic  underline numlist   |  removeformat link bullist  outdent indent |\
+                      styleselect fontsizeselect | table| strikethrough forecolor backcolor |\
+                      subscript superscript charmap  | help',
+  };
+
+  if (!props.canEdit) {
+    return (
+      <>
+        <tr>
+          <td scope="col" colSpan="2" className="task-specific-description">
+            Why this Task is Important
+            <Editor
+              init={editorViewInit}
+              name="why-info"
+              className="why-info form-control"
+              value={props.whyInfo}
+            />
```

Comment:
If you have a hard time configuring the tinyMCE, why not get rid of these `Editor` altogether, since any roles that enters this condition `!props.canEdit` will never need to edit the info anyway?

My thought is just replacing that editor component with:
`<div dangerouslySetInnerHTML={{ __html: props.whyInfo }}></div>`

It will give us this uneditable look shown on the photo below. Then we can just change the css, maybe make it a white background?
![Screenshot 2023-07-15 at 9 18 15 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/67333705/34828f49-434d-43ed-aa8d-099b4185d40f)

The `props.whyInfo` has `<p> </p>` tags around them. That's why I need to use `dangerouslySetInnerHTML` to get rid of those p tags. An alternative to this is just to get rid of `<p> ` using replace() method. Which I think is better if we are use that the whyInfo always only have p tags

Reply:
Just saw the feedback. The methods you suggested is really useful and I didn't know when I submit the PR. As for why not just replacing `<p>` , I was thinking there could be some format left from the person who creates the task. 
However, the tinyMCE will be removed after the new timer deployed, which makes the config for removing tinyMCE no longer important. I will open a new PR with the Prime React Editor ( the new editor) in the future to see if I can fix it well. Thank you for the detailed and clear feedback.